src/schemes/fluid/weakly_compressible_sph/state_equations.jl

```diff
@@ -49,3 +49,49 @@ function inverse_state_equation(state_equation::StateEquationCole, pressure)
 
     return reference_density * tmp^(1 / exponent)
 end
+
+@doc raw"""
+    StateEquationIdealGas(;sound_speed, reference_density, gamma, background_pressure=0.0)
```

Comment:
```suggestion
    StateEquationIdealGas(; sound_speed, reference_density, gamma, background_pressure=0.0)
```
And can you please include `clip_negative_pressure`?